lib/csdl2markdown.js

```diff
@@ -211,7 +211,10 @@ module.exports.csdl2markdown = function (filename, csdl, referenced = {}) {
     lines.push("");
     lines.push("## " + kind + "s");
 
+    var over = {};
```

Comment:
```suggestion
    const over = {};
```
Don't use `var` inside functions, use either `const` or `let`.